widget/embedded/src/services/eventEmitter.ts

```diff
@@ -0,0 +1,11 @@
+import type { Events, WidgetEventEmitterType } from '../types';
+
+import mitt from 'mitt';
+
+export const eventEmitter = mitt<Events>();
+
+// To be exported for Dapps
+export const widgetEventEmitter: WidgetEventEmitterType = {
+  on: eventEmitter.on,
+  off: eventEmitter.off,
+};
```

Comment:
We don't use Type for naming. (by convention)

```suggestion
export const widgetEventEmitter: WidgetEventEmitter = {
  on: eventEmitter.on,
  off: eventEmitter.off,
};
```

Reply:
Fixed